### ISSUE TITLE
GUI Image of Console is Blurry

### ISSUE THREAD

**Issue by AdrianGreenboy** [renode/renode#582]
Hello Everyone! 

I just installed Renode and all the dependencies on Linux Ubuntu 22.04 , but I am having an issue of seeing the console  blurry and weird when I run the program. 

![image](https://github.com/renode/renode/assets/157384353/1e870f6e-9031-41aa-91e1-068c8e5e9df6)

What can I do to fix this issue? I cannot work when I cannot see anything

Thank you for your attention

**Comment by PiotrZierhoffer** [comment]
Hi @AdrianGreenboy, thanks for reporting the issue.

Frankly speaking this is not something we had seen. Can you give more details on:
- window manager/window decorators you're using
- screen resolution, UI scaling, etc
- X vs Wayland
- how do you obtain Renode (source, package, version)
- GTK version
- do I see it right that the widget we draw is actually being drawn outside of the window?

You can use `renode --console` to reuse the terminal rather than our own window, or `--disable-gui` (or `--disable-xwt` if you're not using the very latest Renode version) to rely on network ports instead.

**Comment by AdrianGreenboy** [comment]
Hello! Thanks for the reply! 

I installed RENODE 1.14.0.29506 from git source: https://github.com/renode/renode

I also installed it with all dependencies stated in the github page. 

I solved the problem, the problem is that I didn't enable WSL2 and python3 . I changed, install these dependencies and packages and I restarted the computer and now works without a problem. 

Still thank you for the answer and support! RENODE is a great tool! 

Adrian
